commands/lib/list.go

```diff
@@ -143,6 +143,7 @@ func GetOutputLibrary(lib *libraries.Library) *rpc.Library {
 		cntplat = lib.ContainerPlatform.String()
 	}
 
+	libHeaders, _ := lib.SourceHeaders()
```

Comment:
Why ignore the error here?

Reply:
Uhm, my reasoning was that since it should never happen in theory it's ok to "do it, or return empty". BTW it's probably better to catch and report the error... I'll push a fix in a moment.